src/main/java/blackjack/Player.kt

```diff
@@ -0,0 +1,15 @@
+package blackjack
+
+import blackjack.card.Card
+
+class Player(val name: String, cards: List<Card>) {
+
+    private val _cards: MutableList<Card> = cards.toMutableList()
+
+    val cards: List<Card>
+        get() = _cards
```

Comment:
컬렉션을 외부에 공개해줄 때에는 방어적 복사를 해주세요.
`List<Card>`로 반환해도 실제 구현체는 MutableList이기 때문에 바깥에서 변경이 가능합니다.

```suggestion
    val cards: List<Card>
        get() = _cards.toList()
```